proxy/proxy.go

```diff
@@ -131,6 +170,8 @@ func CreateStriveProxy(listen string, GGStriveAPIURL string, PatchedAPIURL strin
 	r.Use(middleware.Logger)
 	r.Route("/api", func(r chi.Router) {
 		r.HandleFunc("/sys/get_env", proxy.HandleGetEnv)
+		r.HandleFunc("/user/login", proxy.HandleLogin)
```

Comment:
Better to just not register the handles if the feature isn't being used, instead of having a bunch of if-statements in each handler checking if the feature is on.

Reply:
See how `statsSet` is done. It makes it so you don't have to initialize everything if the feature isn't used.

I'll need to standardize this a bit more if more people plan to add handlers.